content/working_with/console/pages/environments-page.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bt_wiki
+title: Environments Page
+category: Console
+draft: false
+abstract: Environments Page Reference
+weight: 140
+aliases: ["/manager_webui/environments-page/", "/working_with/console/environments-page/"]

```

Comment:
There were no such pages, so I think we can remove aliases from here.

That's more general comment, applies to all `aliases` parameter in newly added pages which does not have any equivalent page in the previous versions.

Reply:
Done